internal/plugins/info.csv

```diff
@@ -53,6 +53,7 @@ cohere_embeddings       ,processor ,cohere_embeddings       ,4.37.0  ,enterprise
 command                 ,processor ,command                 ,4.21.0  ,certified  ,n          ,n     ,n
 compress                ,processor ,compress                ,0.0.0   ,certified  ,n          ,y     ,y
 couchbase               ,cache     ,Couchbase               ,4.12.0  ,community  ,n          ,n     ,n
+couchbase               ,output    ,couchbase               ,0.0.0   ,community  ,n          ,n     ,n
```

Comment:
Version needs setting. I'm aware that this is annoying and awkward as we also have the redundant `.Version` method on the docs type. I think going forwards I'm going to suggest we don't bother filling `.Version` and just focus on this CSV file, as we can use the CSV file to hydrate the actual plugin docs.

Reply:
Done